metadata-io/src/main/java/com/linkedin/metadata/timeline/differ/SchemaDiffer.java

```diff
@@ -304,6 +260,121 @@ private static void sortFieldsByPath(SchemaMetadata schemaMetadata) {
     schemaMetadata.setFields(new SchemaFieldArray(schemaFields));
   }
 
+  private static SchemaField findRenamedField(SchemaField curField, List<SchemaField> targetFields, Set<SchemaField> renamedFields) {
+    return targetFields.stream()
+        .filter(schemaField -> isRenamed(curField, schemaField))
+        .filter(field -> !renamedFields.contains(field))
+        .findFirst().orElse(null);
+  }
+
+  private static boolean isRenamed(SchemaField curField, SchemaField schemaField) {
```

Comment:
Where do we check path match?

Reply:
E.g. if field is 

a.b.c

and becomes

a.b.d 

Could we check parent path of a.b?

Reply:
Tags can be empty, description "" for both

Reply:
Covered blanks and empties, and added a check for last index of "." to compare parent fields for a match

Reply:
Thanks!